presto-spi/src/main/java/com/facebook/presto/spi/ConnectorMetadata.java

```diff
@@ -301,4 +302,12 @@ default void revokeTablePrivileges(ConnectorSession session, SchemaTableName tab
     {
         throw new PrestoException(NOT_SUPPORTED, "This connector does not support revokes");
     }
+
+    /**
+     * List table privileges granted to the specified grantee for the tables that have the specified prefix
+     */
+    default List<GrantInfo> listTablePrivileges(ConnectorSession session, SchemaTablePrefix prefix, String grantee)
```

Comment:
I guess until it's actually removed, I have to add this. Got compilation error otherwise because of the invocation of this method in `transaction/LegacyConnectorMetadata.java`.